app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -651,7 +651,11 @@ public void find_new_raw_curve() {
             BgReading latest_entry = BgReading.lastNoSenssor();
             ra = 0;
             rb = 0;
-            rc = latest_entry.age_adjusted_raw_value;
+            if (latest_entry != null) {
```

Comment:
Not sure what you mean by syntactic testing but in our case here a bg value can never have a null calculated value (it defaults to zero) due to the way sqlite handles it. Maybe that is why you dont see it in real life?